ro.obo

```diff
@@ -333,23 +346,30 @@ is_a: GO:0003674 ! molecular_function
 
 [Term]
 id: GO:0005634
+name: cell nucleus
```

Comment:
Not good.

Reply:
why?

Reply:
Because we now have too conflicting labels for `GO:0005634` which means the OBO file is also not parseable. In any case, feel free to ignore, I am sure it was broken for a long time.

Reply:
This is not in the edit file now or coming from any import. However, I see this label as an exact synonym.

Reply:
Sorry, it comes from COB_import.owl

Reply:
Remove the COB_import update.